app/javascript/listings/dashboard/rowElements/actionButtons.jsx

```diff
@@ -4,24 +4,23 @@ import { h } from 'preact';
 const ActionButtons = ({ isDraft, editUrl, deleteConfirmUrl }) => {
   return (
     <div className="listing-row-actions">
-      {/* <a className="dashboard-listing-bump-button cta pill black">BUMP</a> */}
       {isDraft && (
         <a
```

Comment:
For anchor tags that look like buttons, you can still use the button component with the prop `as`, .e.g. `<Button tagName="a" href={editUrl} className="dashboard-listing-edit-button" />`.

Reply:
@nickytonline Does this block this PR or do you think I can make that update in a different PR? I wanted this PR to be focused solely on getting rid of the old `buttons.scss` file and whatever it attempted to apply styles to.